restapi/src/test/java/io/stargate/web/docsapi/service/DocumentServiceTest.java

```diff
@@ -219,30 +216,6 @@ public void convertToBracketedPath() throws IllegalAccessException, InvocationTa
     assertThat(res).isEqualTo("$['']['a']");
   }
 
-  @Test
-  public void leftPadTo6() throws InvocationTargetException, IllegalAccessException {
-    String result = (String) leftPadTo6.invoke(service, "");
-    assertThat(result).isEqualTo("000000");
-
-    result = (String) leftPadTo6.invoke(service, "00");
-    assertThat(result).isEqualTo("000000");
```

Comment:
I think it's worth keeping this test. Even though we use a lib to do the work, the fact that `0` is used for padding it something affected by our code... but of course we do not need as many test cases now.